### ISSUE TITLE
Forecon synthetic still has fullbright

### ISSUE THREAD

**Issue by XxSWAG-MASTER-OVERLORDxX** [RMC-14/RMC-14#8385]
## Description
As with UNMC synth, forecon synthetic should have halfbright and not fullbright as they currently do

**Comment by Ian321** [comment]
Look at me; making PRs before the issues pop up.
